study.md

```diff
@@ -31,7 +32,11 @@ manipulates it by sorting it and then reversing it in place (i.e., modifying the
 reference to the original).
 
 ```ruby
-# your answer here
+array = [23, 56, 3, 7]
```

Comment:
The instructions are to sort the array in place.  `[23, 56, 3, 7].sort!.reverse!`